src/main/java/frc/robot/subsystems/climb/Climb.java

```diff
@@ -1,9 +1,145 @@
 package frc.robot.subsystems.climb;
 
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfigurator;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 import frc.robot.RobotMap;
+import frc.robot.subsystems.arm.ArmCal;
+
+import java.util.Optional;
+import java.util.TreeMap;
+
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.hardware.CANcoder;;
+
 
 public class Climb extends SubsystemBase {
+    private final TalonFX climbTalonLeft = new TalonFX(RobotMap.CLIMBING_LEFT_MOTOR_CAN_ID);
+    private final TalonFX climbTalonRight = new TalonFX(RobotMap.CLIMBING_RIGHT_MOTOR_CAN_ID);
+    private final CANcoder climbAbsoluteEncoder = new CANcoder(RobotMap.CLIMB_ABS_ENCODER_CAN_ID);
+
+    public enum ClimbPosition{
+        PRE_CLIMB_PREP,
+        CLIMBING,
+        STOWED,
+        CLEAR_OF_ARM,
+    }
+
+    private TreeMap<ClimbPosition, Double> climbPositionMap;
+    private ClimbPosition desiredPosition = ClimbPosition.STOWED;
+    private double desiredPositionDegrees = 0.0;
```

Comment:
from what I can see, you're only using this variable in 2 other places: to update the value and in the sendable builder. if so, just delete this and use climbPositionMap.get(desiredPosition) to obtain the desired degrees. This way, there won't be any accidental discrepancies between the position and the degrees